src/telemetry/telemetry.ts

```diff
@@ -57,7 +57,7 @@ class Telemetry implements ITelemetry {
   }
 
   private getInstrumentationKey() {
-    return (window as any).InstrumentationKey || '';
+    return process.env.REACT_APP_INSTRUMENTATION_KEY || '';
```

Comment:
Update this to `(window as any).InstrumentationKey || process.env.REACT_APP_INSTRUMENTATION_KEY`

We always want to use the InstrumentationKey from the window object and only fallback to the key from env if the window object has no instrumentation key.

Reply:
We intend to use the key in the env since the key in the window will make us write to their instance instead of ours

Reply:
The key is ours. We gave it to the portal team so that we can access it through the window object.